{"name": "Registration with Invalid Email", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"C:\\Users\\a90011969\\Playwright-Python\\Playwright-Python\\playwright-behave-env\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\a90011969\\Playwright-Python\\Playwright-Python\\playwright-behave-env\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"steps\\user_login.py\", line 7, in login\n    context.playwright = sync_playwright().start()\n                         ~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\a90011969\\Playwright-Python\\Playwright-Python\\playwright-behave-env\\Lib\\site-packages\\playwright\\sync_api\\_context_manager.py\", line 84, in start\n    return self.__enter__()\n           ~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\a90011969\\Playwright-Python\\Playwright-Python\\playwright-behave-env\\Lib\\site-packages\\playwright\\sync_api\\_context_manager.py\", line 47, in __enter__\n                raise Error(\n    ...<2 lines>...\n                )\n"}, "steps": [{"name": "Given I navigate to the login page", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"C:\\Users\\a90011969\\Playwright-Python\\Playwright-Python\\playwright-behave-env\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\a90011969\\Playwright-Python\\Playwright-Python\\playwright-behave-env\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"steps\\user_login.py\", line 7, in login\n    context.playwright = sync_playwright().start()\n                         ~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\a90011969\\Playwright-Python\\Playwright-Python\\playwright-behave-env\\Lib\\site-packages\\playwright\\sync_api\\_context_manager.py\", line 84, in start\n    return self.__enter__()\n           ~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\a90011969\\Playwright-Python\\Playwright-Python\\playwright-behave-env\\Lib\\site-packages\\playwright\\sync_api\\_context_manager.py\", line 47, in __enter__\n                raise Error(\n    ...<2 lines>...\n                )\n"}, "start": 1739258905582, "stop": 1739258905588}, {"name": "When I enter invalid details", "status": "skipped", "start": 1739258905592, "stop": 1739258905592}, {"name": "And I submit the form", "status": "skipped", "start": 1739258905592, "stop": 1739258905592}, {"name": "Then I should see an error message", "status": "skipped", "start": 1739258905592, "stop": 1739258905592}], "start": 1739258905581, "stop": 1739258905592, "uuid": "39dc7bf5-23ee-4d76-b45d-6787ce61eed2", "historyId": "e6d4705666732890b4bf58f7579c9d7e", "fullName": "User Login: Registration with Invalid Email", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "User Login"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}